packages/lit-dev-content/src/components/litdev-playground-change-guard.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import '@material/mwc-dialog';
+import '@material/mwc-button';
+
+import {html, css, customElement, internalProperty} from 'lit-element';
+import {nothing} from 'lit-html';
+import {PlaygroundConnectedElement} from 'playground-elements/playground-connected-element.js';
+import {
+  BEFORE_CODE_LANGUAGE_CHANGE,
+  setCodeLanguagePreference,
+} from '../code-language-preference.js';
+
+import type {CodeLanguagePreference} from '../code-language-preference.js';
+
+/**
+ * Prompts the user to continue or cancel if the global code language preference
+ * is about to change if the associated Playground has been modified.
```

Comment:
Nit: "is about to change ~if~ __and__ the associated"

Reply:
Thanks! Done.